lms/static/js/Markdown.Editor.js

```diff
@@ -1468,35 +1468,35 @@
                 xPosition += 25;
             };
 
-            buttons.bold = makeButton('wmd-bold-button', gettext('Bold (Ctrl+B)'), '0px', bindCommand('doBold'));
-            buttons.italic = makeButton('wmd-italic-button', gettext('Italic (Ctrl+I)'), '-20px', bindCommand('doItalic'));
+            buttons.bold = makeButton('wmd-bold-button', gettext('Bold (Ctrl+B)'), '0px', bindCommand('doBold'), 0);

```

Comment:
Not sure why this entire file says I made changes (but there is no difference in the before/after...), but this is the relevant part.